tests/chat/test_chat_manager.py

```diff
@@ -12,7 +12,7 @@ async def async_response_gen():
         return type("MockResponse", (), {"async_response_gen": async_response_gen})
 
     async def achat(self, content, chat_history=None):
-        return type("MockResponse", (), {"message": "chat response"})
+        return type("MockResponse", (), {"response": "chat response"})
```

Comment:
how does this impact existing model responses

Reply:
message was not a valid attribute I believe